README.md

```diff
@@ -84,6 +84,8 @@ To switch open the compose docker-compose.yml and edit the vod2pod image section
 ### Optional API Keys
 - `YT_API_KEY`: Set your YouTube API key (works without but the feed is limited to 15)
   - e.g. YT_API_KEY=AIzaSyBTCCEOHm
+- `PROXY`: Set a proxy for yt-dlp requests (optional)
```

Comment:
As this variable is only used for proxy requests, something like this would be a more descriptive name. I also this this is better suited under the `Environment` header
```suggestion
- `YT_REQUEST_PROXY`: Set a proxy for yt-dlp requests (optional)
```